src/components/congrat-modal/congrat-modal.tsx

```diff
@@ -0,0 +1,27 @@
+import { useContext } from 'react';
+import Modal from '../modal/modal';
+import AppContext from '../app-context/app-context';
+import './style.css';
+import ModalTitle from '../../types/modal-title';
+
+type CongratModalProps = {
+  isModalOpen: boolean;
+};
+
+export default function CongratModal({ isModalOpen }: CongratModalProps) {
+  const { handleCongratModalClose } = useContext(AppContext);
```

Comment:
We need to change this context because you might notice when we add a new modal we add a new callback. Maybe the solution should be a bit more universal.

Reply:
Changed the state of modal provider and its handlers to store any state that could be used in app, so this context has also been changed.